src/applications/pact-act/components/PACTActApp.jsx

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 import Breadcrumbs from './Breadcrumbs';
 
 const PACTActApp = ({ children }) => {
+  document.title = 'Learn how the PACT Act may affect you | Veterans Affairs';
```

Comment:
The `<title>` is no longer being dynamically generated, so I moved it to the top of the application.